SonarQube.Common/AnalysisProperties/Property.cs

```diff
@@ -51,7 +51,7 @@ public bool ContainsSensitiveData()
         /// </summary>
         public string AsSonarRunnerArg()
         {
-            return string.Format(System.Globalization.CultureInfo.InvariantCulture, "-D{0}={1}", this.Id, ProcessRunnerArguments.GetQuotedArg(this.Value));
+            return string.Format(System.Globalization.CultureInfo.InvariantCulture, "-D{0}={1}", this.Id, this.Value);
```

Comment:
Turns out that passing args to an Exe and to a Bat can be done safely by just quoting the arg and doubling any existing quotes. No need for special handling here.